sdk/storage/Azure.Storage.Blobs/src/AppendBlobClient.cs

```diff
@@ -108,6 +108,7 @@ public AppendBlobClient(string connectionString, string blobContainerName, strin
         /// A <see cref="Uri"/> referencing the append blob that includes the
         /// name of the account, the name of the container, and the name of
         /// the blob.
+        /// This is likely to be similar to "https://{yourstorageaccountname}.blob.core.windows.net/{yourstoragecontainername}/{pathtoblob}".
```

Comment:
`pathtoblob` is kind of misleading.  You can create blob names that look like paths, but they aren't paths in a meaningful sense.  I think it'd be better to just use `.../{container_name}/{blob_name}`.  It'll also relate better to the other .ctors which have `containerName` and `blobName` parameters.

Reply:
Also removing the your* prefix and adding underscores as @tg-msft suggests makes the comment more readable